README.md

```diff
@@ -12,7 +12,7 @@ If you are interested in contributing to the development of OSCAL, refer to the
 
 ## Project Status
 
-OSCAL [1.0.1](https://github.com/usnistgov/OSCAL/releases) was released on January 30th, 2022. There are also [release notes](https://pages.nist.gov/OSCAL/reference/release-notes/) containing a summary of changes in this and previous releases.
+The latest release of OSCAL is [1.0.3](https://github.com/usnistgov/OSCAL/releases) which was released on May 9th, 2022. There are also [release notes](https://pages.nist.gov/OSCAL/reference/release-notes/) containing a summary of changes in this and previous releases.
```

Comment:
> @aj-stein-nist yep, let me know once a decision has been made.

I got a chance to talk to @david-waltermire-nist  (sorry, confused the order of meetings 😆) and he agrees we ought to just point to the release notes page. We do not need to explicitly call out the version in this `README.md` file as it is yet another thing we have to tweak during the release. (The others we will automate, but not this one in the short-term.) Can you adjust it accordingly and I re-review?